x/ccv/provider/keeper/proposal.go

```diff
@@ -217,6 +219,8 @@ func (k Keeper) StopConsumerChain(ctx sdk.Context, chainID string, closeChan boo
 	k.DeleteValidatorSetCap(ctx, chainID)
 	k.DeleteAllowlist(ctx, chainID)
 	k.DeleteDenylist(ctx, chainID)
+	k.DeleteAllowInactiveValidators(ctx, chainID)
+	k.DeleteMinStake(ctx, chainID)
```

Comment:
nit: move `DeleteMinStake` before `DeleteAllowInactiveValidators` so it's in the same order as in the `HandleConsumerModificationProposal`.